src/Transition/CreateNewIDFUsingRulesV23_2_0.f90

```diff
@@ -317,16 +317,17 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                   CoilLatentStuff(NumCoilLatentStuff)%ParentType = 'AIRLOOPHVAC:UNITARYSYSTEM'
                   CoilLatentStuff(NumCoilLatentStuff)%ParentName = MakeUPPERCase(IDFRecords(Num)%Alphas(1))
                   CoilLatentStuff(NumCoilLatentStuff)%NewCurveName = TRIM(IDFRecords(Num)%Alphas(1)) // "-AutogeneratedPLFCurve"
+
                   CoilLatentStuff(NumCoilLatentStuff)%HeatingCoilType = MakeUPPERCase(IDFRecords(Num)%Alphas(12))
                   IF (CoilLatentStuff(NumCoilLatentStuff)%HeatingCoilType == 'COIL:HEATING:WATERTOAIRHEATPUMP:PARAMETERESTIMATION') CoilLatentStuff(NumCoilLatentStuff)%ActuallyCreateCurve = .true.
                   IF (CoilLatentStuff(NumCoilLatentStuff)%HeatingCoilType == 'COIL:HEATING:WATERTOAIRHEATPUMP:EQUATIONFIT') CoilLatentStuff(NumCoilLatentStuff)%ActuallyCreateCurve = .true.
                   CoilLatentStuff(NumCoilLatentStuff)%HeatingCoilName = MakeUPPERCase(IDFRecords(Num)%Alphas(13))
+
                   CoilLatentStuff(NumCoilLatentStuff)%CoolingCoilType = MakeUPPERCase(IDFRecords(Num)%Alphas(14))
-                  IF (CoilLatentStuff(NumCoilLatentStuff)%CoolingCoilType == 'COIL:COOLING:DX:VARIABLESPEED') CoilLatentStuff(NumCoilLatentStuff)%ActuallyCreateCurve = .true.
```

Comment:
Do not create the PLF curve if it's not going to be used. The only types (4 in total) are 

* `Coil:<Cooling/Heating>:WaterToAirHeatPump:EquationFit`
* `Coil:<Cooling/Heating>:WaterToAirHeatPump:ParameterEstimation`

Reply:
@mjwitte this coil type should have a PLF curve?

    Coil:Cooling:DX:VariableSpeed,
      A4,  \field Energy Part Load Fraction Curve Name
        \required-field
        \type object-list
        \object-list UnivariateFunctions
        \note quadratic curve = a + b*PLR + c*PLR**2
        \note cubic curve = a + b*PLR + c*PLR**2 + d*PLR**3
        \note PLR = part load ratio (cooling load/steady state capacity)